content/applications/sales/members.rst

```diff
@@ -0,0 +1,91 @@
+=======
+Members
+=======
+
+The *Members* application is where all operations related to memberships can be configured and
+managed. The *Members* app integrates with the *Sales* and *Accounting* applications to sell and
+invoice memberships directly to customers.
+
+Membership products
+===================
+
+To create a new membership product, navigate to :menuselection:`Members app --> Configuration -->
+Membership Products`, and click :guilabel:`New` to open a blank product record.
+
+Complete the blank form with the necessary information, including the :guilabel:`Membership
+Duration`.
```

Comment:
I think this needs to be expanded upon, and provide the reader with information about all the fields present on the form. Doing so would provide some much-needed context, and add some value to the overall doc.